src/main/java/com/gamegoo/service/member/BlockService.java

```diff
@@ -29,7 +29,7 @@ public class BlockService {
     private final ChatCommandService chatCommandService;
     private final FriendService friendService;
 
-    Integer pageSize = 9;
+    private final static Integer PAGE_SIZE = 10;
```

Comment:
NotificationService.java 처럼 
```suggestion
    private static final Integer PAGE_SIZE = 10;
```
로 통일하셔도 좋을 것 같습니다!